src/controllers/index.ts

```diff
@@ -7,6 +7,8 @@ export * as beneficialOwnerStatements from "./beneficial.owner.statements.contro
 export * as beneficialOwnerType from "./beneficial.owner.type.controller";
 export * as checkYourAnswers from "./check.your.answers.controller";
 export * as landing from "./landing.controller";
+export * as updateLanding from "./update.landing.controller";
+export * as overseasEntitySearch from "./overseas.entity.search.controller";
```

Comment:
wondering if search is the correct term here. Maybe Query instead of Search. This would have multiple similar knock on changes if we go with that change.

Reply:
Yeah I can rename/refactor that. Is the concern that search is maybe a bit generic?
retrieveOverseasEntity
getOverseasEntity
overseasEntityQuery
Happy to do any

Reply:
overseasEntityQuery gets my vote

Reply:
done!